src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyTextFilterTest.java

```diff
@@ -0,0 +1,604 @@
+    }
+
+    /**
+     * Performs verification of the file with the given file path using specified configuration
+     * and the array of expected messages. Also performs verification of the config with filters
+     * specified in the input file.
+     * This method is an exact copy of
+     * {@link AbstractModuleTestSupport#verifyWithInlineConfigParser} and the purpose of it is
+     * to be able to call {@link #verifyViolationsSortedActualViolations}. This is a temporary fix
+     * until sorting is implemented in {@link AbstractModuleTestSupport#verifyViolations}.
+     *
+     * @param filePath file path to verify.
+     * @param expectedUnfiltered an array of expected unfiltered config.
+     * @param expectedFiltered an array of expected config with filters.
+     * @throws Exception if exception occurs during verification process.
+     * @noinspection JavadocReference
+     * @noinspectionreason This method is temporary.
+     */
+    @SuppressForbidden
+    private void verifyFilterWithInlineConfigParserSortedActualViolations(String filePath,
+                                                            String[] expectedUnfiltered,
+                                                            String... expectedFiltered)
+            throws Exception {
+        final TestInputConfiguration testInputConfiguration =
+                InlineConfigParser.parseWithFilteredViolations(filePath);
+        final DefaultConfiguration configWithoutFilters =
+                testInputConfiguration.createConfigurationWithoutFilters();
+        final List<TestInputViolation> violationsWithoutFilters =
+                new ArrayList<>(testInputConfiguration.getViolations());
+        violationsWithoutFilters.addAll(testInputConfiguration.getFilteredViolations());
+        Collections.sort(violationsWithoutFilters);
+        verifyViolationsSortedActualViolations(configWithoutFilters,
+                filePath, violationsWithoutFilters);
+        verify(configWithoutFilters, filePath, expectedUnfiltered);
+        final DefaultConfiguration configWithFilters =
+                testInputConfiguration.createConfiguration();
+        verifyViolationsSortedActualViolations(configWithFilters,
+                filePath, testInputConfiguration.getViolations());
+        verify(configWithFilters, filePath, expectedFiltered);
+    }
+
+    /**
+     * Performs verification of violation lines.
+     * This method is a copy of {@link AbstractModuleTestSupport#verifyViolations}
+     * apart from the sorting of the actual violations.
+     *
+     * @param config parsed config.
+     * @param file file path.
+     * @param testInputViolations List of TestInputViolation objects.
+     * @throws Exception if exception occurs during verification process.
+     * @noinspection JavadocReference
+     * @noinspectionreason This method is temporary.
+     */
+    public void verifyViolationsSortedActualViolations(Configuration config,
```

Comment:
move to abstract class to reuse by others

Reply:
Same here.

Reply:
It will take few years for new contributor to finish migration. Better to place this method at right place on first commit

Reply:
This method would not work with other modules. `Collections.sort(..` creates problems with other modules because it sorts strings and other modules that have violations such as `100: xxxx`, `101: xxx` run into problem because violations with line 100 and above get sorted first, which shouldn't be the case. 

There is no need to introduce new method and and migrate everything when current one can be extended without causing disruptions.

Reply:
Might take a bit of effort to find a way to sort strings first based on line number and then on the violation message, but that's it.

Reply:
Nobody will use your new methods until required. All should work as before.